dev-lisp/sbcl/sbcl-2.5.4.ebuild

```diff
@@ -50,7 +50,10 @@ CDEPEND=">=dev-lisp/asdf-3.3:= \
 BDEPEND="${CDEPEND}
 		dev-debug/strace
 		doc? ( sys-apps/texinfo >=media-gfx/graphviz-2.26.0 )
-		system-bootstrap? ( || ( dev-lisp/clisp dev-lisp/sbcl ) )"
+		system-bootstrap? ( || (
+			!elibc_musl? ( dev-lisp/clisp )
```

Comment:
If clisp doesn't work on musl, it should be masked there. If it's already masked, then having it inside of `|| ( a b c )` will work fine, i.e. there shouldn't be a need for `elibc_musl` conditionals here.

Reply:
Do you want the update to the musl profile masking that in this pull request as a separate commit?

Reply:
Yes please.